src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss

```diff
@@ -89,69 +93,133 @@ $layoutSwitchResolution: 1100px;
 }
 
 .title {
-  font-family: $font-gramatika-black;
-  @include hl-font-size-title(30px);
-  line-height: 1.3;
-  width: 40%;
+  max-width: 320px;
+
+  & > h2 {
+    font-family: $font-gramatika-black;
+    @include hl-font-size-title(30px);
+    line-height: 45px;
+  }
+
   @media screen and (max-width: $layoutSwitchResolution) {
-    width: 100%;
-    @include hl-margin(bottom, 40px);
+    max-width: 100%;
   }
 }
 
 .inputWrapper {
   display: flex;
   align-items: center;
+  position: relative;
   @include hl-width(360px);
-  @include hl-padding(top, 5px);
-  @include hl-padding(bottom, 36px);
-  border-bottom: 1px solid $white-color;
+  @include hl-padding(top, 28px);
+  @include hl-padding(bottom, 15px);
+  border-bottom: 2px solid $white-color;
+
+  & > label {
+    position: absolute;
+    top: 9px;
+    left: 0;
+    display: inline-block;
+    font-family: 'Gramatika', sans-serif;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: bold;
+    color: white;
+    opacity: 0;
+    transition: all 0.3s ease-in-out;
+  }
+
+  @media screen and (max-width: $layoutSwitchResolution) {
+    max-width: 100%;
+    width: 100%;
+    padding-top: 18px;
+    padding-bottom: 10px;
+
+    & > label {
+      top: 0;
+    }
+  }
   @media screen and (max-width: $sm) {
     width: 100%;
-    padding: 0;
   }
 }
+
 .input {
+  font-family: 'Sofia-Pro', sans-serif;
   background: transparent;
   border: none;
-  @include hl-font-size-text(16px);
+  @include hl-font-size-text(18px);
+  line-height: 30px;
+  font-weight: 300;
   color: $white-color;
-  letter-spacing: 6px;
-  text-transform: uppercase;
+  letter-spacing: 0.5px;
   width: 100%;
-  padding-right: 10px;
+  padding: 0 10px 0 0;
+
   &::placeholder {
     @include hl-font-size-text(14px);
-    line-height: 1.7;
-    letter-spacing: 10px;
-    text-transform: uppercase;
+    letter-spacing: 0.5px;
     opacity: 0.5;
     transition: opacity 0.2s ease-out;
   }
 
-  &:hover {
+  &:focus {
     &::placeholder {
-      opacity: 0.8;
+      opacity: 0;
+    }
+
+    & ~ label {
+      opacity: 0.5;
     }
   }
 }
 
 .inputWrapper.error {
-  border-bottom: 1px solid $red-on-blue-color;
+  border-bottom: 2px solid $red-on-blue-color;
 }
 
 .button {
   border: none;
   background: none;
   padding: 5px 0 0;
   cursor: pointer;
+  font-size: 0;
+  line-height: 0;
+
+  & > svg {
```

Comment:
svg is the only place where it's ok to use tag selectors

Reply:
fixed